mozci/task.py

```diff
@@ -368,6 +368,16 @@ class GroupSummary(RunnableSummary):
     def __post_init__(self):
         assert all(self.name in t.groups for t in self.tasks)
 
+        # Because of https://bugzilla.mozilla.org/show_bug.cgi?id=1640758, we can't trust
+        # test-verify when there are no reported failures.
+        # TODO: Drop this filtering a few months after https://bugzilla.mozilla.org/show_bug.cgi?id=1640758 is fixed.
```

Comment:
nit: no need to put url in comment twice.

Reply:
Fixed